### ISSUE TITLE
Add custom file path to series added from "This Week"

### ISSUE THREAD

**Issue by DaveAlton** [mylar3/mylar3#1428]
**Describe the request**
I want to use the "This Week" functionality, but I use very custom file paths. I don't have them set automatically. 
When I add a series through search, I can set a path. I want to do the same for the "This Week" feature. This would solve my issues and I wouldn't have to wait until ComicVine finally updated their site with info. 

**How did you come up with the idea?**
From seeing how this tool works. I can set a custom file path from search, but I can through "This Week"

**Comment by evilhero** [comment]
Adding a path via the weekly page wouldn't allow it to download anything until CV has the information. All it would be able to do is create the folder and add the series to your watchlist, it wouldn't be able to download anything until it knows it exists on CV. Anything on the pullist has to have a corresponding CV entry before it knows it exists for a series. 

What your asking for, I believe at least, is outside the scope of what mylar actually is intended for - to watch for particular issues for a series and download them when available. 

It sounds like you just want to mark issues for download individually without adhering to the series watchlist/being on CV aspect, which is the only way mylar works.

**Comment by DaveAlton** [comment]
No, that’s not what I’m asking for. What I want is to watch for specific series and add them when they’re available. Mylar already does this. The only change is that I want the ability to tell Mylar which directory to use when it starts watching. Then, when the series is available, and Mylar adds it, it also sets the directory I’ve given it.   On Tue, Oct 3, 2023 at 10:36 PM, evilhero ***@***.***> wrote:  
Adding a path via the weekly page wouldn't allow it to download anything until CV has the information. All it would be able to do is create the folder and add the series to your watchlist, it wouldn't be able to download anything until it knows it exists on CV. Anything on the pullist has to have a corresponding CV entry before it knows it exists for a series.
What your asking for, I believe at least, is outside the scope of what mylar actually is intended for - to watch for particular issues for a series and download them when available.
It sounds like you just want to mark issues for download individually without adhering to the series watchlist/being on CV aspect, which is the only way mylar works.
